### DISCUSSION TITLE
I just have one quick question about the hack because I'm confused on if i have to wait

### DISCUSSION THREAD

**Discussion by Endernates** [3kh0/ext-remover#204] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I was trying to unblock stuff on a school computer until I found out that you can get banned from electronics. But on the discussion titled Just wait are you saying that I should wait a month to not get banned from electronics?

**Comment by 3kh0** [comment, marked as answer]
Depends on your school, some schools will not care while others will punish you for doing any of these hacks.

**Reply by Endernates** [reply]
can the school arrest me? 

> Depending on your school, some schools will not care while others will punish you for doing any of these hacks.

Can the school arrest me and if they can How likely is it?

**Reply by 3kh0** [reply]
@Endernates yes they can, it depends on your school

**Comment by ghost** [comment]
It's illegal, no matter what school. Not being noticed doesn't make it any more legal. Basically, you can only use bookmarklets/ltbeef legally

**Reply by synapt0** [reply]
what about proxies, are those illegal?

**Reply by ghost** [reply]
No, proxies are legal. If you really, really need one, search up holy unblocker (aio) on github, and deploy on replit. (CIRCUMVENT THE TOS FIRST! You can get in serious trouble if you don't know what you are doing) https://github.com/holy-unblocker/website-aio/wiki/Circumventing-deployment-restrictions

**Reply by synapt0** [reply]
good
